plugins/dataTypes/Boolean/Boolean.class.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @package DataTypes
+ */
+
+class DataType_Boolean extends DataTypePlugin {
+
+	protected $isEnabled = true;
+	protected $dataTypeName = "Boolean";
+	protected $hasHelpDialog = true;
+	protected $dataTypeFieldGroup = "numeric";
+	protected $dataTypeFieldGroupOrder = 11;
+	protected $jsModules = array("Boolean.js");
+
+
+	/**
+	 * @param string $runtimeContext "generation" or "ui"
+	 */
+	public function __construct($runtimeContext) {
+		parent::__construct($runtimeContext);
+		if ($runtimeContext == "generation") {		
+		}
+	}
```

Comment:
I used some other DataType as basis to create this one.
I will remove this function.